### PR TITLE
Update README and POM file

### PR DESCRIPTION
**Pull request by anamatasanch**
+Upade README
+Update project ID on pom.xml file

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>amatasanchez-sps-summer20</deploy.projectId>
           <deploy.version>1</deploy.version>
         </configuration>
       </plugin>
```
Comment (TsueiCat) [inline portfolio/pom.xml:41]: The changes made for the pull request should reflect the title. So in this case, all the commits should relate to updating the README file. The other commits made should be placed in a separate pull request under a title summarizing those changes.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,20 @@
...
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Ana Mata's Portfolio</h1>
+      <p>I'm a junior at The University of Texas at El Paso majoring in Computer science.
+          My interests include web development and databases. 
+      </p>
+      <img src="Images/ana.jpg" style="height:50vh" alt="Ana Mata">
+      <p>Click here to get a random anime recomendation:</p>
       <button onclick="addRandomGreeting()">Hello</button>
```
Comment (TsueiCat) [inline portfolio/src/main/webapp/index.html]: It looks like what this button does is give anime recommendations? It might be better to rename the button text to something better like 'Click here for anime recommendations!' and get rid of the paragraph text. Same thing with updating the function name.

File: portfolio/src/main/webapp/js/materialize.js [hunk 3]
```diff
@@ -0,0 +1,12374 @@
...
+      htmlMatch = /<.+>/,
+      singlet = /^\w+$/;
+
+  function find(selector, context) {
+    context = context || doc;
+    var elems = classMatch.test(selector) ? context.getElementsByClassName(selector.slice(1)) : singlet.test(selector) ? context.getElementsByTagName(selector) : context.querySelectorAll(selector);
+    return elems;
+  }
+
+  var frag;
```
Comment (TsueiCat) [inline portfolio/src/main/webapp/js/materialize.js]: Try to avoid abbreviated variable names; it helps people reading your code to better understand when names are as descriptive as possible to leave little room for confusion

File: portfolio/pom.xml (hunk 1 above)
Comment (anamatasanch) [reply portfolio/pom.xml:41]: I renamed the pull request for now. Is it possible to create multiple pull requests for a branch or should I move the changes to another branch next time?

**Review by TsueiCat — CHANGES_REQUESTED** [review]
Wow!! You really went all out :) Skimming the commits, I'd suggest updating variable and method names to better describe what their function is (i.e. no abbreviations).

File: portfolio/pom.xml (hunk 1 above)
Comment (TsueiCat) [reply portfolio/pom.xml:41]: I think the best way to go about it is to create a branch of this branch and make your changes there. Wait for the first branch's pull request to go through and then merge changes into the second branch. Then, create a pull request of the second branch. Ideally, the pull request should be short so that the review time doesn't take too long :)

File: portfolio/src/main/webapp/js/materialize.js (hunk 3 above)
Comment (anamatasanch) [reply portfolio/src/main/webapp/js/materialize.js]: Sadly, this is not my code. This is the materialize framework. It's a really handy tool, I like it better than bootstrap because it's almost pure html and css. https://materializecss.com/

Comment (TsueiCat) [reply portfolio/src/main/webapp/js/materialize.js]: Ohhh, interesting ok. Didn't know about this thanks!

File: portfolio/pom.xml (hunk 1 above)
Comment (joanapascoal) [reply portfolio/pom.xml:41]: I agree. Multiple branches are the simplest way to keep the work parallelized, but it can be a little confusing and even tricky when the merge conflict appears. So if you feel that the flow isn't clear just reach us and we can share some tips.

Ideally this PR, should be split into 2: 
- One PR that only contains the README.md . Title suggestion: "Update the README.md file to describe the repo."
- One other with the basic structure, explaining in the description exactly what is being changed.  e.g. just adding the unchanged project skeleton or replacing all the string by something, etc

I'm not requesting to split this into 2 different PR at this point, but I'm sharing with you how you could do that, since it's possible that you'll need that in the future.

There is multiples ways to split an already existing PR in multiple ones, but I suggest:
1- git cherry-pick -n (cherry-pick changes from commit without committing them yet)
2- git reset HEAD (unstage everything)
3- git add [file_path] (stage the specified files)
4- git clean -f (remove all untracked files, the ones not selected in the previous step)
5- git commit

**Comment by joanapascoal** [comment]
Hey! Really nice to see a few PR coming over :)

As few generic tips, I suggest take a look at:
https://sites.google.com/sps-program.com/students/technical-reference-guides/code-reviews/code-review-checklist

**Comment by joanapascoal** [comment]
> _No description provided._

Can you confirm what is being updated in this PR in description?
For what I see, you are doing:
**week-0** 
- "Modify README"

**week-1**
- Add a small description about you.

Is this correct or am I missing something?

**Comment by anamatasanch** [comment]
> > _No description provided._
> 
> Can you confirm what is being updated in this PR in description?
> For what I see, you are doing:
> **week-0**
> 
> * "Modify README"
> 
> **week-1**
> 
> * Add a small description about you.
> 
> Is this correct or am I missing something?

This is correct. I'll update it asap.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (anamatasanch) [reply portfolio/src/main/webapp/index.html]: I have decided to move this code to the other PR. will fix on that branch!

**Review by joanapascoal — APPROVED** [review]
Really nice job splitting and cleaning this! The scope now is super clear.

**Comment by joanapascoal** [comment]
> Wow!! You really went all out :) Skimming the commits, I'd suggest updating variable and method names to better describe what their function is (i.e. no abbreviations).

@TsueiCat I believe that this comment now is out of the scope and should be considered in the following PR. Can you check?

**Review by TsueiCat — APPROVED** [review]
